pep-0668.rst

```diff
@@ -0,0 +1,1099 @@
+package in an interpreter-wide context, it can do so in a manner that
+will avoid conflicting with the external package manager and reduces
+the risk of breaking software shipped by the external package manager.
+
+Terminology
+===========
+
+A few terms used in this PEP have multiple meanings in the contexts
+that it spans. For clarity, this PEP uses the following terms in
+specific ways:
+
+distro
+    Short for "distribution," a collection of various sorts of
+    software, ideally designed to work properly together, including
+    (in contexts relevant to this document) the Python interpreter
+    itself, software written in Python, and software written in other
+    languages. That is, this is the sense used in phrases such as
+    "Linux distro" or "Berkeley Software Distribution."
+
+    A distro can be an operating system (OS) of its own, such as
+    Debian, Fedora, or FreeBSD. It can also be an overlay distribution
+    that installs on top of an existing OS, such as Homebrew or
+    MacPorts.
+
+    This document uses the short term "distro," because the term
+    "distribution" has another meaning in Python packaging contexts: a
+    source or binary distribution package of a single piece of Python
+    language software, that is, in the sense of
+    ``setuptools.dist.Distribution`` or "sdist". To avoid confusion,
+    this document does not use the plain term "distribution" at all.
+    In the Python packaging sense, it uses the full phrase
+    "distribution package" or just "package" (see below).
+
+    The provider of a distro - the team or company that collects and
+    publishes the software and makes any needed modifications - is its
+    **distributor**.
+package
+    A unit of software that can be installed and used within Python.
+    That is, this refers to what Python-specific packaging tools tend
+    to call a "`distribution package`_" or simply a "distribution";
+    the colloquial abbreviation "package" is used in the sense of the
+    Python Package Index.
+
+    .. _`distribution package`: https://packaging.python.org/glossary/#term-Distribution-Package
+
+    This document does not use "package" in the sense of an importable
+    name that contains Python modules, though in many cases, a
+    distribution package consists of a single importable package of
+    the same name.
+
+    This document generally does not use the term "package" to refer
+    to units of installation by a distro's package manager (such as
+    ``.deb`` or ``.rpm`` files). When needed, it uses phrasing such as
+    "a distro's package." (Again, in many cases, a Python package is
+    shipped inside a distro's package named something like ``python-``
+    plus the Python package name.)
+Python-specific package manager
+    A tool for installing, upgrading, and/or removing Python packages
+    in a manner that conforms to Python packaging standards (such as
+    PEP 376 [#PEP-376]_ and PEP 427 [#PEP-427]_). The most popular
```

Comment:
There’s a syntax for referencing PEPs

```suggestion
    :pep:`376` :pep:`427`). The most popular
```

(links in the footnote need to also be removed.

Reply:
I'm following the advice in PEP 12 about how to write PEP links, which suggests doing it this way. Happy to update that advice if it's wrong :) (And I guess I'm already ignoring the advice about two spaces after periods and about the Emacs incantation)

Reply:
You don't have to explicitly link PEPs; tooling does that for you.

Reply:
Aha, I just tried out the Sphinx build which does that, so I switched to just "PEP 376 and PEP 427".